news/other-4117.md

```diff
@@ -0,0 +1 @@
+docker image: Nightly production docker images are now available as `balabit/syslog-ng:nightly`
```

Comment:
This is good news, I'll add it to our nightly announcement if you don't mind.

Reply:
Sure, thanks :)